src/components/organisms/AssetContent/EditHistory.tsx

```diff
@@ -4,52 +4,56 @@ import { useAsset } from '../../../providers/Asset'
 import EtherscanLink from '../../atoms/EtherscanLink'
 import Time from '../../atoms/Time'
 import styles from './EditHistory.module.css'
+import { gql, useQuery } from '@apollo/client'
 
+const getReceipts = gql`
+  query ReceiptData($address: String!) {
+    datatokens(where: { id: $address }) {
+      createTime
+      tx
+      updates(orderBy: timestamp, orderDirection: desc) {
+        id
+        tx
+        timestamp
+      }
+    }
+  }
+`
 interface Receipt {
```

Comment:
we should remove this custom interface and rely on the generated Apollo typings.

Tested this import:

```js
import { ReceiptData_datatokens_updates as ReceiptData } from '../../../@types/apollo/ReceiptData'
```

Then use `ReceiptData` as the type in the code. And with that we already catch a minor issue which is the point of the typings:

<img width="461" alt="Screen Shot 2021-03-04 at 17 53 37" src="https://user-images.githubusercontent.com/90316/109999481-8f2e6d80-7d12-11eb-8aa6-731cc4077237.png">

Reply:
ok, on it